test/test.js

```diff
@@ -498,4 +481,16 @@ describe('Routes', () => {
       assert.strictEqual(body.attestation, retrieval.attestation)
     })
   })
+
+  async function givenRetrieval (props = {}) {
+    const createRequest = await fetch(
+    `${spark}/retrievals`,
+    {
+      method: 'POST',
+      body: JSON.stringify({ sparkVersion, ...props })
+    }
```

Comment:
```suggestion
      `${spark}/retrievals`,
      {
        method: 'POST',
        body: JSON.stringify({ sparkVersion, ...props })
      }
```

Reply:
Good catch.

I wish our linter would catch this automatically.